react/features/chat/components/native/Chat.js

```diff
@@ -42,8 +41,9 @@ class Chat extends AbstractChat<Props> {
                     behavior = 'padding'
                     style = { styles.chatContainer }>
                     <Header>
-                        <BackButton onPress = { this.props._onToggleChat } />
-                        <HeaderLabel labelKey = 'chat.title' />
+                        <HeaderNavigation
```

Comment:
Hum, this looks a bit odd. Wouldn't it mke sense to have a `HeaderWithNavigation` component which extends header adding the buttons, instead of needing to nest the components?